prSHARK/backends/github.py

```diff
@@ -336,14 +336,14 @@ def parse_pr_list(self, prs):
             new_pr.is_locked = pr['locked']
             new_pr.lock_reason = pr['active_lock_reason']
             new_pr.is_draft = pr['draft']
-            new_pr.created_at = dateutil.parser.parse(pr['created_at'])
-            new_pr.updated_at = dateutil.parser.parse(pr['updated_at'])
+            new_pr.created_at = dateutil.parser.parse(pr['created_at']).replace(tzinfo=None)
```

Comment:
Could you shortly explain why the change is needed? Is the timezone info missing from the API?

Reply:
when we save the PR the timezone information will not be saved in MongoDB. So when we compare new_pr and mongo_pr it will show a difference. For example : {'new_value': datetime.datetime(2011, 1, 26, 19, 1, 12, tzinfo=tzutc()), 'old_value': datetime.datetime(2011, 1, 26, 19, 1, 12)}

Reply:
Alright, in that case we should just check whether the API returns UTC so that we can remove the timezone and save it to the DB. The important thing is that we have UTC in the DB.

Reply:
Yes, we will have UTC time in the DB.  The statement replace(tzinfo=None) will only remove the info.

Reply:
You can probably add a small datetime helper function here. My suggestion is analogous to the actionSHARK.

```suggestion
            new_pr.created_at = dateutil.parser.parse(pr['created_at']).astimezone(pytz.UTC).replace(tzinfo=None)
```